### PR TITLE
Feature/quotes container

### PR DESCRIPTION
**Pull request by Njoe00**
Review code for quotes container and background colour in tailwind config. Branch checkout from main so code base is old.

### CODE REVIEW

File: src/components/QuotesContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import QuotesText from "./QuotesText";
+import Image from "next/image";
```
Comment (MKCMMSK) [inline src/components/QuotesContainer.tsx]: I would sort external dependencies from internal depedencies

File: src/components/QuotesContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            alt="alt"
+          />
+          {quotesText}
+        </div>
+      </div>
+    </div>
+  );
+}
+
+const Quotes = [
```
Comment (MKCMMSK) [inline src/components/QuotesContainer.tsx]: strange that this is capitalized

File: src/components/QuotesContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  {
+    title: "/logo-gear-patrol-white.png",
+    quote:
+      "“Bellroy can take a backbreaker of a wallet and make it smaller than a deck of cards.”",
+    imageDimensions: [164.44, 30],
+  },
+  {
+    title: "/mens-journal-white.avif",
+    quote: "“Seriously slim wallets and passport holders.”",
```
Comment (MKCMMSK) [inline src/components/QuotesContainer.tsx:40]: both of these quotes are meant to have extra quotes to be displayed?

File: src/components/QuotesContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+export default function QuotesContainer() {
+  const [quotesText, setQuotesText] = useState(Quotes[0].quote);
+  const [buttonHighlighted, setButtonHighLighted] = useState(0);
+
+  return (
+    <div className="bg-background-quotes flex justify-center p-20">
+      <div className="flex items-center justify-center flex-col text-gray-200 font-serif">
+        <h1 className="text-2xl flex py-6">Don't just take our word for it</h1>
+        <QuotesText
+          Quotes={Quotes}
```
Comment (MKCMMSK) [inline src/components/QuotesContainer.tsx]: seems odd that this is caplitalized, seems like an anti pattern

File: src/components/QuotesText.tsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+}: QuoteTextProps) {
+  const handleHighlightedButtons = (index: number) => {
+    setButtonHighLighted(index);
+  };
+
+  return (
+    <div className="flex-row flex items-center line-block ">
+      {Quotes.map((text, index) => {
+        return (
+          <>
```
Comment (MKCMMSK) [inline src/components/QuotesText.tsx]: seems like a useless fragment

File: src/components/QuotesText.tsx [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+            <span
+              className={
+                buttonHighlighted === index
+                  ? "hover:text-white text-white text-center text-xl border-b-2 inline-block py-10 px-[10px] w-[280px] h-[132px]"
+                  : "text-gray-400 border-gray-400 text-center hover:border-white border-b-2 text-xl inline-block py-10 px-[10px] w-[280px] h-[132px]"
```
Comment (MKCMMSK) [inline src/components/QuotesText.tsx:32]: There is duplication here that's not necessary. Whether it's the selected button right now or not, you're still setting certain classes regardless. py-10, px-[10px] etc

File: src/components/QuotesContainer.tsx [hunk 7]
```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import QuotesText from "./QuotesText";
+import Image from "next/image";
+
+export default function QuotesContainer() {
+  const [quotesText, setQuotesText] = useState(Quotes[0].quote);
+  const [buttonHighlighted, setButtonHighLighted] = useState(0);
```
Comment (MKCMMSK) [inline src/components/QuotesContainer.tsx]: Can't you move this into quotes text?

File: src/pages/index.tsx [hunk 8]
```diff
@@ -412,7 +417,7 @@ export default function Home() {
 
                 <div className="flex flex-col w-[413.33px] h-[508px] items-center relative">
                   <div className="top-10 relative">
-                    <Image src={cardImage} height={300} width={300} alt="alt" />
+                    <Image src={`${cardImage || content.image[0]}`}  height={300} width={300} alt="alt" />
```
Comment (MKCMMSK) [inline src/pages/index.tsx]: why is this in a template literal?

**Review by MKCMMSK — CHANGES_REQUESTED** [review]
Looks better

File: src/components/QuotesContainer.tsx (hunk 1 above)
Comment (Njoe00) [reply src/components/QuotesContainer.tsx]: addressed.

File: src/components/QuotesContainer.tsx (hunk 2 above)
Comment (Njoe00) [reply src/components/QuotesContainer.tsx]: addressed.

File: src/components/QuotesContainer.tsx (hunk 3 above)
Comment (Njoe00) [reply src/components/QuotesContainer.tsx:40]: Yes, its apart of the styling of the website.

File: src/components/QuotesContainer.tsx (hunk 4 above)
Comment (Njoe00) [reply src/components/QuotesContainer.tsx]: addressed for the entire PR.

File: src/components/QuotesContainer.tsx (hunk 7 above)
Comment (Njoe00) [reply src/components/QuotesContainer.tsx]: addressed.

File: src/components/QuotesText.tsx (hunk 5 above)
Comment (Njoe00) [reply src/components/QuotesText.tsx]: turned into a div.

File: src/pages/index.tsx (hunk 8 above)
Comment (Njoe00) [reply src/pages/index.tsx]: addressed in a previous pull request.

File: src/components/QuotesText.tsx (hunk 6 above)
Comment (Njoe00) [reply src/components/QuotesText.tsx:32]: this does not appear to be the case, when I remove styling from either side it will affect the styling.

**Pull request merged**